fastf1/events.py

```diff
@@ -388,6 +388,32 @@ def get_event_schedule(year, *, include_testing=True, force_ergast=False):
     return schedule
 
 
+def get_events_remaining(dt=None, *, include_testing=True, force_ergast=False) -> 'EventSchedule':
+    """Create an :class:`~fastf1.events.EventSchedule` object for remaining season.
+
+    Args:
+        dt (datetime): Optional DateTime to get events after.
+        include_testing (bool): Include or exclude testing sessions from the
+            event schedule.
+        force_ergast (bool): Always use data from the ergast database to
+            create the event schedule
+
+    Returns:
+        :class:`~fastf1.events.EventSchedule`
+
+    .. versionadded:: 2.2
+    """
+    if dt is None:
+        dt = datetime.datetime.now()
+
+    events = _get_schedule(dt.year, include_testing=include_testing, force_ergast=force_ergast)
+    result = []
+    for index, row in events.iterrows():
+        if dt < row["EventDate"]:
+            result.append(row)
+
+    return result
```

Comment:
The return type here is `list` and not `EventSchedule`. This needs to be fixed.
You could also probably just use pandas indexing instead of looping over the whole object. Something along the lines of `events = events.loc[events['EventDate'] >= dt]` (haven't tested this)

Reply:
Thank you! Great advice that worked out of the box.